src/installing-from-source.rst

```diff
@@ -28,11 +25,33 @@ Checking dependencies
 ---------------------
 
 You need:
+- `Git <https://git-scm.com/downloads>`_
+- `Python 3.6 or 3.7 <https://www.python.org/downloads/>`_ (3.8+ not supported, see :gh-aw:`this comment <activitywatch/issues/433#issuecomment-653397090>`), check with :code:`python3 -V` (required to build the core components)
+- `Poetry <https://python-poetry.org/docs/#installation>`_, check with :code:`poetry -V` (can be installed like this: :code:`python3 -m pip install poetry`)
+- `Node 12 or higher <https://www.npmjs.com/get-npm>`_, check with :code:`node -v` and :code:`npm -v` (required to build the web UI)
+- `Rust nightly and cargo <https://doc.rust-lang.org/cargo/getting-started/installation.html>`_, nightly can be installed using the following commands:
+
+.. code-block:: sh
+
+   rustup install nightly
+   rustup default nightly
+
+check with :code:`rustc -V` and :code:`cargo -V` (for building aw-server-rust)
+
+**For Windows users:**
+You also need:
```

Comment:
It's also worth noting that you need Git Bash.
It's possible to install git without Git Bash.

Reply:
Yes, I know. But I tried to build the project with the normal cmd.exe and it worked too.

Reply:
Huh, interesting. Well then it's OK.